presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/EnvironmentList.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.tests.product.launcher.cli;
+
+import com.google.common.collect.ImmutableList;
+import com.google.inject.Module;
+import io.airlift.airline.Command;
+import io.airlift.log.Logger;
+import io.prestosql.tests.product.launcher.Extensions;
+import io.prestosql.tests.product.launcher.LauncherModule;
+import io.prestosql.tests.product.launcher.env.EnvironmentFactory;
+import io.prestosql.tests.product.launcher.env.EnvironmentModule;
+import io.prestosql.tests.product.launcher.env.EnvironmentOptions;
+
+import javax.inject.Inject;
+
+import static io.prestosql.tests.product.launcher.cli.Commands.runCommand;
+import static java.util.Objects.requireNonNull;
+
+@Command(name = "list", description = "lists environments")
+public final class EnvironmentList
+        implements Runnable
+{
+    private final Module additionalEnvironments;
+    private static final Logger log = Logger.get(EnvironmentList.class.getSimpleName());
+
+    @Inject
+    public EnvironmentOptions environmentOptions = new EnvironmentOptions();
+
+    public EnvironmentList(Extensions extensions)
+    {
+        this.additionalEnvironments = requireNonNull(extensions, "extensions is null").getAdditionalEnvironments();
+    }
+
+    @Override
+    public void run()
+    {
+        runCommand(
+                ImmutableList.<Module>builder()
+                        .add(new LauncherModule())
+                        .add(new EnvironmentModule(additionalEnvironments))
+                        .add(environmentOptions.toModule())
+                        .build(),
+                EnvironmentList.Execution.class);
+    }
+
+    public static class Execution
+            implements Runnable
+    {
+        private EnvironmentFactory factory;
+
+        @Inject
+        public Execution(EnvironmentFactory factory)
+        {
+            this.factory = requireNonNull(factory, "factory is null");
+        }
+
+        @Override
+        public void run()
+        {
+            log.info("Available environments: ");
```

Comment:
shouldn't this be displayed on `stdout`? logging is used for diagnostic, and here the `list` command goal is to display this information. Notice that `log.properties` should not be able to affect this command behaviour.

Reply:
good point. 

may be useful
https://github.com/prestosql/presto/blob/cd65c0552816d36309ec24b8c0817831a49afdfd/presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/testcontainers/PrintingLogConsumer.java#L43-L44

Reply:
Done but I don't know how to supress standard airlift logging.